docs/api/background-tasks.mdx

```diff
@@ -0,0 +1,106 @@
+---
+sidebar_position: 6
+title: Background APIs
+description: "Understand APIs that run in the background"
+---
+
+# Background APIs (EAP)
+
+:::info Early Access Program
+Background APIs are part of the Early Access Program (EAP) and are subject to change.
+:::
+
+Some APIs in Permit run in the background, and can be triggered without awaiting the response.
+These APIs are commonly used for operations that require scheduling, are not time-sensitive, or may take significant time to complete.
+
+For example, we will use the Copy Environment API to explain how to use background APIs.
+
+```bash
+curl --location 'https://api.permit.io/v2/projects/default/envs/staging/copy/async' \
+  -H 'authorization: Bearer API_SECRET_KEY'\
+  --data '{
+  "target_env": {
+    "new": {
+        "key": "prod",
+        "name": "production"
+    }
+  }
+}'
+# Response: 202 Accepted
+```
+When triggering a Copy Environment, the API will return a `202 Accepted` response, indicating that the job has been accepted and is being processed in the background.
+
+The response contains info about the task, including the `task_id` that can be used to check the status of the task.
```

Comment:
Explain here how to check status (or at least add link)

Reply:
There is an example how to check one line after